code/obj/machinery/clonepod.dm

```diff
@@ -1152,3 +1152,28 @@
 #undef DEFAULT_MEAT_USED_PER_TICK
 #undef DEFAULT_SPEED_BONUS
 #undef MEAT_LOW_LEVEL
+
+/obj/machinery/clonepod/automatic
+	name = "Cloning Pod Deluxe"
+	meat_level = 1.#INF
+
+/obj/machinery/clonepod/automatic/process()
+	if(!src.attempting)
+		spawn_a_ghost()
+	return..()
+
+/obj/machinery/clonepod/automatic/New()
+	..()
+	animate_rainbow_glow(src) // rgb shit cause it looks cool
+	SubscribeToProcess()
+
+/obj/machinery/clonepod/automatic/disposing()
+	..()
+	UnsubscribeProcess()
+
+/obj/machinery/clonepod/automatic/proc/spawn_a_ghost()
+	for(var/mob/dead/observer/ghost in mobs)
+		var/datum/mind/ghost_mind = ghost.mind
+		if(ghost.client && !ghost_mind.dnr)
+			spawn(growclone(ghost, ghost.real_name, ghost_mind))
```

Comment:
Not sure why the spawn is written this way. Idk if a spawn is necessary though, and i don't think it's exacuting it in paralell like spawns do.